index.html

```diff
@@ -1,11 +1,244 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/ac0885c8f6.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="style.css">
     <title>Embedding Videos and Images</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,500;1,400&display=swap"
+        rel="stylesheet">
 </head>
+
 <body>
-    
+    <nav class="nav-layout">
+        <div class="logo">
+            <div class="menu-icon">
+                <i class="fas fa-bars"></i>
+            </div>
+            <div class="youtube-icon">
+                <img src="./images/youtube-logo.png" alt="youtube icon" />
+            </div>
+        </div>
+
+        <div class="search">
+            <input type="search" placeholder="Search" />
+            <button type="submit"><i class="fas fa-search"></i></button>
+        </div>
+
+        <div class="notify">
+            <div class="upload">
+                <img src="./images/upload.png" alt="upload file icon">
+            </div>
+            <div class="sign-in">
+                <button>Sign In</button>
+            </div>
+            <div class="notify-bell">
+                <i class="fas fa-bell"></i>
+            </div>
+        </div>
+    </nav>
+    <main class="bottom-section">
+        <section class="left-side">
+
+            
+                <iframe  style="width: 100%; border: none; height: 500px;" src="https://www.youtube.com/embed/wyj17UwXkbU"
+                    allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+                    allowfullscreen></iframe>
+            
+
+            <div class="video-meta-wrapper">
+                <div class="video-title">
+                    <h1>Microverse Coding Challenges</h1>
+                </div>
+                <div class="video-stats-wrapper">
+                    <div class="left-stats">
+
+                        <div class="thumbnail">
+                        </div>
+
+                        <div class="chanel-name-sub-wrapper">
+                            <div class="channel-name">
+                                Microverse
+                            </div>
+                            <div class="subscribe-button">
+                                <span class="red-part">
+                                    <i class="fab fa-youtube"></i>
+                                    Subscribe
+                                </span>
```

Comment:
- [ ] **[Optional]** You named it subscribe-button but you didn't use a button tag ![image](https://user-images.githubusercontent.com/53256139/87077129-460c3500-c1e8-11ea-8178-cbaa5f6b996e.png)